MANIFEST.in

```diff
@@ -8,11 +8,7 @@ include extlib/CMakeLists.txt
 include extlib/nlohmann/json.hpp
 
 # fmt
-recursive-include extlib/fmt/include *
-recursive-include extlib/fmt/src *
-recursive-include extlib/fmt/support/cmake *.cmake
-include extlib/fmt/CMakeLists.txt
-include extlib/fmt/*.rst
+recursive-include extlib/fmt *
```

Comment:
why do we have to include things like `extlib/fmt/test/`?
Including more made the sdist size increase by almost 50%.

Reply:
Try to reduce it.